src/data/faq.json

```diff
@@ -484,7 +484,8 @@
                         "anchor": "other_platforms",
                         "active": true,
                         "textblock": [
-                            "It's not foreseeable at the moment if and when the app will be available in other app stores except Google or Apple. The app will use the Apple and Google Exposure Notification APIs. The community is invited to contribute and use the app for other APIs."
+                            "It's not foreseeable at the moment if and when the app will officially be available in other app stores except Google or Apple. The app will use the Apple and Google Exposure Notification Framework. The community is invited to contribute and use the app for other frameworks.",
+                            "For F-Droid, there exists an inofficial fork of the app under the name <a href='https://codeberg.org/corona-contact-tracing-germany/cwa-android' target='_blank' rel='noopener noreferrer'>Corona Contact Tracing Germany</a> which is maintained by the community. Because the development of the fork is not supervised by the Robert Koch-Institut, we can give no official support for problems with the app published on F-Droid."

```

Comment:
Change "Robert Koch-Institut" to English name "Robert Koch Institute" or use abbreviation "RKI" from above suggestion.

Reply:
fixed